src/utils/self_test/SConscript

```diff
@@ -9,24 +9,39 @@ import os
 import daos_build
 
 SELF_TEST = 'self_test.c'
+LIB_CONFIG_INI = 'src/configini.c'
+LIB_CONFIG_CNF = 'etc/selftest.cnf'
+LIB_CONFIG_ECNF = 'etc/selftest_expected.cnf'
+
+# Function to filter out gcc flags from this compilation env
+def remove_certain_gcc_flags(str):
+    import re
```

Comment:
Move this `import re` to the top of the file (near `import os`).

Reply:
Moved to after import os.